src/components/DpButtonRow/DpButtonRow.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    :class="[`text--${align}`, $attrs.class]"
+    :class="[`text-${align}`, $attrs.class]"
```

Comment:
Constructing the classes in such a way will stop tailwind from discovering them when parsing the source files. See https://tailwindcss.com/docs/content-configuration#dynamic-class-names